local-volume/README.md

```diff
@@ -83,6 +83,13 @@ $ ALLOW_PRIVILEGED=true LOG_LEVEL=5 FEATURE_GATES=PersistentLocalVolumes=true ha
 3. Continue with [Running the external static provisioner](#running-the-external-static-provisioner)
    below.
 
+
+#### GCE E2E Testing
+``` console
+KUBE_FEATURE_GATES="PersistentLocalVolumes=true" NODE_LOCAL_SSDS=1 go run hack/e2e.go -- -v --up
```

Comment:
No, it only works on GCE.  But the details of bringing up a cluster with local disks for each environment are already covered up above.  So I think this entire line can be removed, and just keep the 2nd line that actually runs the tests.